app/controllers/standup_meetings_controller.rb

```diff
@@ -0,0 +1,16 @@
+class StandupMeetingsController < ApplicationController
+  before_action :set_standup_meeting_group
+
+  def edit
+    @standup_meeting = StandupMeeting.find(params[:id])
+    authorize @standup_meeting
+  rescue Pundit::NotAuthorizedError
+    redirect_back_or_to(standup_meeting_groups_path)
```

Comment:
new in updated PR.

Reply:
how come we are doing it like this??

Reply:
@toyhammered updated and ready for rereview!